test/tools/yulopti.cpp

```diff
@@ -87,32 +88,68 @@ class YulOpti
 		}.printErrorInformation(_errors);
 	}
 
-	void parse(string const& _input)
+	shared_ptr<Object> getSubObject(shared_ptr<Object> const& _object, string const& _qualifiedPath)
+	{
+		if (_qualifiedPath.empty() || _qualifiedPath == _object->name.str())
+			return _object;
+
+		yulAssert(
+			boost::algorithm::starts_with(_qualifiedPath, _object->name.str() + "."),
+			"Assembly object not found."
+		);
+
+		const auto subObjectPath = _qualifiedPath.substr(_object->name.str().length() + 1);
+		const auto subObjectName = subObjectPath.substr(0, subObjectPath.find_first_of('.'));
+
+		auto subObjectIt = find_if(
+			_object->subObjects.begin(),
+			_object->subObjects.end(),
+			[subObjectName](auto const& subObject) { return subObject->name.str() == subObjectName; }
```

Comment:
```suggestion
			[subObjectName](auto const& _subObject) { return _subObject->name.str() == subObjectName; }
```
Our code style is to use underscore for parameter variable names.

Reply:
Done